sal_python_pkg/sal/utils.py

```diff
@@ -93,7 +94,15 @@ def serializer(obj):
     if isinstance(obj, datetime.datetime):
         # Make sure everything has been set to offset 0 / UTC time.
         obj = obj.astimezone(datetime.timezone.utc).isoformat()
-    return obj
+        return obj
```

Comment:
I would just compact the two lines here into `return obj.astimezone...`

Reply:
Removed the redundant obj lines. If you want me to change anything else let me know.